consoleapp.py

```diff
@@ -5,6 +5,11 @@
 
 import re
 from consolestyle import fclr, style
+try:
+    import readline
+except:
+    pass
+
```

Comment:
```suggestion
try:
    import readline
except:
    pass
```
example for a suggestion

Reply:
Note that you can't suggest a change when there is no line in the near that has been changed in the PR.
That's a feature that I don't know why but github does not allow. It's really annoying.